### PR TITLE
List Structure Implementation for Todo App

### PR DESCRIPTION
**Pull request by AyoMoses1**
# List Structure Review
I implemented the list structure of the to-do list app following the best guidelines for HTML, CSS, and JavaScript. I also used webpack to bundle my project files. Please feel free to review and drop your feedback where necessary

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const task = document.createElement('li');
+  task.innerHTML = `
+                   <div>
+                    <input type="checkbox" id="check" name="task_check">
+                    <label for="task_check">${arr.description}</label>
+                   </div>
+                    <i class="fa-solid fa-ellipsis-vertical"></i>  
+                  `;
+  listPlaceholder.appendChild(task);
+});
```
Comment (zieeco) [inline src/index.js]: - Hey, great job, however, according to the project requirement, you're meant to use a function to display the todos as you've used a loop, kindly wrap the loop in a function to meet this part of the project requirement.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+ul.list__placeholder li input {
+  margin-right: 1.2rem;
+  cursor: pointer;
+}
+
+ul.list__placeholder li i {
+  cursor: pointer;
+}
```
Comment (zieeco) [inline src/style.css]: - Hey, great job with the styling, however, you're design is not looking close to the design from the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Please update your styles to look like the design in the video above. See the images below to compare your design:



*design on video*
Screenshot from 2023-01-26 15-41-35

*your design*
Screenshot from 2023-01-26 15-40-28

File: README.md [hunk 3]
```diff
@@ -1 +1,216 @@
...
+
+I would like to thank Cindy Shin for the awesome and inspiring design idea of this project. The design guidelines for this project was gotten from one of her most profound designs for a conference. Please find here a link to the <a href="https://creativecommons.org/licenses/by-nc/4.0/legalcode">license</a>, <a href="https://creativecommons.org/licenses/by-nc/4.0/#">disclaimer</a> and a link to the original <a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015">design</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
```
Comment (zieeco) [inline README.md]: - Hey, Please, kindly remove all the placeholder text in the readme as these are instructions meant for you to follow in order to add content to your readme and they're not meant to be in the readme after you've updated the content on your readme. please, see them below to remove them.


Screenshot from 2023-01-26 15-50-11
Screenshot from 2023-01-26 15-50-00
Screenshot from 2023-01-26 15-49-41

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AyoMoses1 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit message. ✔️



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**